windows/src/desktop/kmshell/locale/zh-CN/strings.xml

```diff
@@ -0,0 +1,912 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  strings.xml: Contains all localizable strings for Keyman for Windows
+  * Create a user interface translation for Keyman for Windows at https://translate.keyman.com/
+-->
+<resources>
+  <!-- Context: System -->
+  <!-- String Type: PlainText -->
+  <!-- Introduced: 7.1.266.0 -->
+  <string name="S_LocaleAuthors" comment="Name(s) of people who created this translation - i.e. your name">Keyman</string>
+  <!-- Context: _Font -->
+  <!-- String Type: PlainText -->
+  <!-- Introduced: 7.0.230.0 -->
+  <string name="SK_UIFontName" comment="The standard font">Segoe UI</string>
+  <!-- Context: _Font -->
+  <!-- String Type: PlainText -->
+  <!-- Introduced: 7.0.230.0 -->
+  <string name="SK_UIFontSize" comment="The standard font size">9</string>
+  <!-- Context: Formatted Messages -->
+  <!-- String Type: FormatString -->
+  <!-- Introduced: 7.0.230.0 -->
+  <string name="SKButtonYes" comment="Yes button in message boxes">&amp;是</string>
```

Comment:
These probably need to be more like: 

```suggestion
  <string name="SKButtonYes" comment="Yes button in message boxes">是 (&amp;Y)</string>
```

Because there is no <kbd>是</kbd> key that we can press for the shortcut. <kbd>Alt</kbd>+<kbd>Y</kbd> is a normal pattern we see elsewhere. For example, see this Windows UI with <kbd>Alt</kbd>+<kbd>C</kbd> for the button:

![image](https://user-images.githubusercontent.com/4498365/133165456-a9f6cf53-5403-4674-a72b-843af31d8801.png)